decidim-comments/app/packs/src/decidim/comments/comments.component_for_testing.js

```diff
@@ -1,4 +1,5 @@
 /* eslint no-undef: 0 */
+/* eslint-disable no-relative-import-paths/no-relative-import-paths */
```

Comment:
This is the only ignore that I needed to add because if this isn't relative the tests fail. I think we can live with this as it's used only for testing infraestructure.